ckanext/switzerland/templates/showcase/add_datasets.html

```diff
@@ -0,0 +1,88 @@
+{% extends 'showcase/edit_base.html' %}
+
+{% block subtitle %}{{ _('Applications - Add datasets') }}{% endblock %}
+
+{% block wrapper_class %} ckanext-showcase-edit-wrapper{% endblock %}
+
+{% block ckanext_showcase_edit_span %}span12{% endblock %}
+
+{% block primary_content_inner %}
+  <section class="module">
+    <div class="module-content">
+      {% block form %}
+        {% set facets = {
+          'fields': c.fields_grouped,
+          'search': c.search_facets,
+          'titles': c.facet_titles,
+          'translated_fields': c.translated_fields,
+          'remove_field': c.remove_field }
+        %}
+        {% set sorting = [
+          (_('Relevance'), 'score desc, metadata_modified desc'),
+          (_('Name Ascending'), 'title_string asc'),
+          (_('Name Descending'), 'title_string desc'),
+          (_('Last Modified'), 'metadata_modified desc'),
+          (_('Popular'), 'views_recent desc') if g.tracking_enabled else (false, false) ]
+        %}
+        {% snippet 'snippets/search_form.html', type='dataset', query=c.q, sorting=sorting, sorting_selected=c.sort_by_selected, count=c.page.item_count, facets=facets, show_empty=request.params, error=c.query_error, fields=c.fields %}
+      {% endblock %}
+       <h3 class="page-heading">
+        {% block page_heading %}
+            {{ _('Datasets available to add to this application') }}
+        {% endblock %}
+      </h3>
+      {% block package_search_results_list %}
+        {% if c.page.items %}
+          <form method="POST" data-module="basic-form">
+            {#{% block errors %}{{ form.errors(error_summary) }}{% endblock %}#}
+            <table class="table table-bordered table-header table-hover table-bulk-edit table-edit-hover" data-module="table-selectable-rows">
+              <col width="8">
+              <col width="120">
+              <thead>
+                <tr>
+                  <th></th>
+                  <th class="table-actions">
+                    <div class="btn-group">
+                      <button name="bulk_action.showcase_add" value="add" class="btn" type="submit">
+                        <i class="fa fa-plus icon-plus"></i>
+                        {{ _('Add to Application') }}
```

Comment:
Here application is capitalized. At other places it is not. I think it should be uniform. But since this the backend it might not be super important. I am usure about where to capitalize myself.

Reply:
I agree with you, it's really a mess how the capitalisation varies in the showcase extension! I didn't change anything about that for now, though, as I only wanted to get the new translations in.